dashboard/src/components/api/codenvy-organization-events-manager.factory.ts

```diff
@@ -157,22 +157,22 @@ export class CodenvyTeamEventsManager {
    *
    * @param handler new team handler function
    */
-  addNewTeamHandler(handler: Function): void {
-    this.newTeamHandlers.push(handler);
+  addNewOrganizationHandler(handler: Function): void {
+    this.newOrganizationHandlers.push(handler);
   }
 
   /**
    * Process team renamed event.
    *
    * @param info
    */
-  processRenameTeam(info: any): void {
+  processRenameOrganization(info: any): void {
     let isCurrentUser = this.isCurrentUser(info.initiator);
     if (isCurrentUser) {
       //TODO
     } else {
-      let title = 'Team renamed';
-      let content = 'Team \"' + info.oldName + '\" has been renamed to \"' + info.newName + '\" by ' + info.initiator;
+      let title = 'Organization/Team renamed';
```

Comment:
-1 on message format "**Organization/Team** test has been renamed ..."
@slemeur do we need live notifications for organizations in onprem?